department-of-veterans-affairs/va.gov-cms
# VAGOV-3348 menu links et

**QA:** 

- As an admin, login, and visually verify that the node types found in screenshot in https://va-gov.atlassian.net/browse/VAGOV-3348 are available to Pittsburgh menu
- Visually verify that these content types can all be hidden from menu when exclusion button under the menu add (in sidebar) is selected - will require visiting the Pittsburgh menu to confirm item is disabled after saving.
